### DISCUSSION TITLE
ZuluSCSI 1.2 and Adaptec 29160 Controller

### DISCUSSION THREAD

**Discussion by McGrude** [ZuluSCSI/ZuluSCSI-firmware#419] [General]
I connected my ZuluSCSI 1.2 to my Adaptec 29160 controller and was unable to successfully install DOS 6.22 or Windows98 on it -- claims there's no fixed disks attached.   In the Adaptec controller BIOS ( hit control-A during boot when shown ) it does see the ZuluSCSI ( in either disk image mode or direct raw mode , and I can low-level format and verify the device(s).   The direct raw or disk images show correctly in the SCSI ID summary as I expect with the correct IDs and sizes.   When booting from the Windows98SE CDrom (from IDE cd drive) it claims there are no fixed disks attached.  When I boot a DOS 6.22 floppy it claims the same.   I connected a Sun2.1 hard disk drive to see if it was some issue with the Adaptec BIOS but Windows 98SE installer was happy to fdisk/format that drive.   How can I diagnose this odd behavior?

**Comment by dranch** [comment]
Did you install the ASPI drivers for DOS (via config.sts) or offer the Adaptec ASPI drivers at the initial Win98 install stage?

**Comment by aperezbios** [comment]
@McGrude as @dranch stated, you do need to have suitable Win98/DOS ASPI drivers installed, and that's almost surely the problem here.  That said, there are custom-made Windows installers that have such drivers slip-streamed in. As long as your Adaptec controller card sees the SCSI device, it's almost certain that you've got a driver issue.

**Comment by McGrude** [comment]
Thank you both.   This was definitely operator error.  It's been a while since I installed Windows98SE... :)

After the ASPI driver clue I created a Windows98SE boot floppy, updated the ASPI8U2.SYS file for the Adaptec U160 controller and I'm now formatting a 32G SD card in direct/raw mode as I type this.  Thank you for the clue I needed to get past this hurdle.
